.lando.yml

```diff
@@ -42,6 +42,7 @@ services:
     type: elasticsearch:custom
     overrides:
       image: bitnami/elasticsearch:7.8.0
+    portforward: 13306
```

Comment:
Hehehe, you set the same port as MySql 🙃

```suggestion
    portforward: 19200
```

Reply:
Copy pasta!